Python/doubly.py

```diff
@@ -0,0 +1,77 @@
+"""Doubly linked list implementation"""
+
+class Node:
+    """
+    Class for node.
+
+    'list' for reference to the linked list, 'data' for storing data,
+    'prev' for pointing on previous node, 'next' - for the next one
+    """
+    def __init__(self, lst, data):
+        self.lst = lst
+        self.data = data
+        self.prev = None
+        self.next = None
+
+
+class LinkedList:
```

Comment:
Can you please implement at least `__len__` and `__getitem__`? A custom implementation of the iterable protocol would be awesome too.